scripts/xsslint_thresholds.json

```diff
@@ -28,5 +28,5 @@
         "python-wrap-html": 226,
         "underscore-not-escaped": 507
     },
-    "total": 1770
```

Comment:
@efischer19 
1. Can you just bump the total to 1771, which is the number of violations (see below):
2. You need to find the individual rule that is passing the threshold (see below again), and also bump that by one.

The following is the report from master failing:
```
Number of xss_linter.py violations: 1771

javascript-concat-html violations: 160
javascript-escape violations: 7
javascript-interpolate violations: 29
javascript-jquery-append violations: 77
javascript-jquery-html violations: 189
javascript-jquery-insert-into-target violations: 23
javascript-jquery-insertion violations: 19
javascript-jquery-prepend violations: 7
mako-html-entities violations: 0
mako-invalid-html-filter violations: 11
mako-invalid-js-filter violations: 192
mako-js-html-string violations: 0
mako-js-missing-quotes violations: 0
mako-missing-default violations: 182
mako-multiple-page-tags violations: 0
mako-unknown-context violations: 0
mako-unparseable-expression violations: 0
mako-unwanted-html-filter violations: 0
python-close-before-format violations: 0
python-concat-html violations: 24
python-custom-escape violations: 13
python-deprecated-display-name violations: 41
python-interpolate-html violations: 64
python-parse-error violations: 0
python-requires-html-or-text violations: 0
python-wrap-html violations: 226
underscore-not-escaped violations: 507
```

Reply:
roger 👍